tap_shiphero/sync.py

```diff
@@ -198,6 +227,9 @@ def sync_daily(client, catalog, state, start_date, end_date, stream_id, stream_c
     to_col = stream_config['to_col']
     records_fn = stream_config['get_records']
 
+    # Set the page to start paginating on
+    params['page'] = get_bookmark(state, stream_id, 'page', 1)
```

Comment:
This addition corresponds to the deletion of `page = 1` on line 139 of the original file